generator/src/test/java/com/graphicsfuzz/generator/semanticschanging/Expr2BinaryMutationFinderTest.java

```diff
@@ -80,11 +80,26 @@ public int nextInt(int bound) {
         return 2;
       }
 
+      @Override
+      public int nextInt(int origin, int bound) {
+        return IRandom.super.nextInt(origin, bound);
```

Comment:
Presumably this is never called as the original test didn't depend on it. So I suggest replacing the body with:

```
throw new RuntimeException("This method was added to the IRandom interface after this test was written and thus should not be called; if it becomes required the IRandom implementation in the test should be revised.");
```